From eac2ad495b29f15d78daa2a7226653f36515cd7a Mon Sep 17 00:00:00 2001
From: David Gibson <david@gibson.dropbear.id.au>
Date: Mon, 25 Mar 2019 14:52:47 +1100
Subject: [PATCH] Update version.lds again

Yet again, we've added several functions to libfdt that were supposed
to be exported, but forgotten to add them to the versio.lds script.
This adds them.

Signed-off-by: David Gibson <david@gibson.dropbear.id.au>
---
 libfdt/version.lds | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/libfdt/version.lds b/libfdt/version.lds
index 9f5d708..a5fe62d 100644
--- a/libfdt/version.lds
+++ b/libfdt/version.lds
@@ -66,6 +66,10 @@ LIBFDT_1.2 {
 		fdt_resize;
 		fdt_overlay_apply;
 		fdt_get_string;
+		fdt_get_max_phandle;
+		fdt_check_full;
+		fdt_setprop_placeholder;
+		fdt_property_placeholder;
 	local:
 		*;
 };
